### ISSUE TITLE
Improve the check command view

### ISSUE THREAD

**Issue by mohierf** [shinken-monitoring/mod-webui#726]
On the host/service view, the check command is simply rendered as the check command name. There is no information about the arguments and about the real command that is executed after the macro expansion.

This may be improved like this:
![image](https://user-images.githubusercontent.com/2932687/71183233-ea904f00-2277-11ea-9136-1644d1290682.png)

**Comment by mohierf** [comment]
@maethor : if you wish to backport, you can cherry-pick this commit: https://github.com/mohierf/mod-webui/commit/ddf162a855cfe765bf2dab84414b3338aafe4ffa

**Comment by mohierf** [comment]
This rendering is not really perfect because it would be better to display the real executed command. At the moment, in the example, the `check_mongodb` command is not expanded before the macro expansion. To be completed...

**Comment by mohierf** [comment]
Something like this content looks more useful:
![image](https://user-images.githubusercontent.com/2932687/71185810-70ae9480-227c-11ea-8bce-09ede8fc3fa2.png)

The real executed command and its macro expansion 👍 

Not yet perfect because some macros are not expanded ... but it looks better. Work in progress...

**Comment by mohierf** [comment]
The most recent commit I pushed is a squash of 3 commits I did to test and improve the macro expansion in the commands. The final result is:
![image](https://user-images.githubusercontent.com/2932687/71319269-2f94cb00-249c-11ea-8128-4ac479bdfcd4.png)

**Comment by maethor** [comment]
Nice idea. I will have a look.

**Comment by mohierf** [comment]
Indeed, on my own fork, the integ branch is up-to-date with many modifications and I tested as much as I could with Alignak AND Shinken (thanks to docker) ... you should give a try and we could merge in the main repo ?

**Comment by maethor** [comment]
Would you be able to make PR ? I prefer to avoid cherry picking. In this issue for instance, I don't know if the commit you are talking about in the second message is covering the modifications you are talking about in the forth and fifth messages.
